### PR TITLE
Add end-of-year blog post for SPOC Community

### PR DESCRIPTION
**Pull request by nomadscientist**
Big thanks to @hexylena for the community training metrics! That's awesome!

### CODE REVIEW

File: news/images/spoc_images/2024_spoc.png
Comment (hexylena) [inline news/images/spoc_images/2024_spoc.png:1]: is there maybe a more relevant community pic? maybe all y'all on one of the hackathon zooms?

Comment (nomadscientist) [reply news/images/spoc_images/2024_spoc.png:1]: erm...

I have this but it's not brilliant

**Comment by nomadscientist** [comment]
Screenshot 2024-12-18 at 19 00 51
It's not rendering on Codespace though, something is wrong

File: news/images/spoc_images/2024_spoc.png
Comment (hexylena) [reply news/images/spoc_images/2024_spoc.png:1]: no, but it's maybe a bit more human? idk. I'd maybe rather see real people for something so community focused.

Comment (nomadscientist) [reply news/images/spoc_images/2024_spoc.png:1]: fair enough :)

File: news/_posts/2024-12-18-spoc.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  authorship:
+    - nomadscientist
+    - pavanvidem
+tags:
+  - gtn
+  - communications
+  - single-cell
+layout: news
+cover: "news/images/spoc_images/2024_spoc.png"
+coveralt: "A group of people standing along a staircase holding up the Vulcan salute forSPOC: 'Live long and prosper'. They look relatively happy. Taken at 2024 Galaxy Community Conference."
```
Comment (bgruening) [inline news/_posts/2024-12-18-spoc.md:13]:
```suggestion
coveralt: "A group of people standing along a staircase holding up the Vulcan salute for SPOC: 'Live long and prosper'. They look relatively happy. Taken at 2024 Galaxy Community Conference."
```

**Review by shiltemann — APPROVED** [review]

**Pull request merged**